xrtpy/__init__.py

```diff
@@ -9,7 +9,7 @@
 from xrtpy import response
 
 try:
-    from xrtpy.version import __version__
+    from xrtpy.version import __version__  # noqa
```

Comment:
The `# noqa` comment (short for "no quality assurance") means that `flake8`, `ruff`, and other linters won't autofix or raise errors about this line.

Often `# noqa` is followed by an error code for a specific error, i.e. `# noqa: TRY003`, so that it does all other checks/autofixes but that one.  I didn't add it here because there's a `ruff` rule which I was planning on adding later that would automatically do this for us.